app/views/shelter_reviews/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<section class = "info">
+  <h2>Create Review:</h2>
+  <% if !flash[:error].nil? %>
+  <div class = "error">
+    <% flash[:error].each do |msg| %>
+      <div>
+          <%= msg %>
+      </div>
+    <% end %>  
+  </div>
```

Comment:
This looks good, but I think we should opt for `<section>` instead of `<div>` as it's more explicit.

Reply:
Agreed